netket/models/jastrow.py

```diff
@@ -42,7 +42,14 @@ def __call__(self, x_in: Array):
             "kernel", self.kernel_init, (nv * (nv - 1) // 2,), self.param_dtype
         )
 
-        W = jnp.zeros((nv, nv), dtype=self.param_dtype).at[il].set(kernel)
+        ## separate real and imaginray part in order to be effificient on GPUs
```

Comment:
```suggestion
        # .at[].set is VERY slow for complex128 numbers in jax.
        # So we do it on the real-valued real and imaginary parts separately and then join them back
        # See issue https://github.com/jax-ml/jax/issues/24872
```

Reply:
@lindamauron out of curiosity, can you also benchmark having `jnp.complex64` parameters, is there a difference?